### ISSUE TITLE
Error  when resize window

### ISSUE THREAD

**Issue by houzw** [Flyer53/jsPanel4#58]
Encounter an error:
```
Uncaught TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.
    at e.onwindowresize.window.addEventListener.t (jspanel.min.js?58c3:1)
```
use` jspanel4/es6module,`
browser is Chrome

**Comment by Flyer53** [comment]
@houzw 
Thanks for the report.
I'm aware of this issue (which appears only in some situations as far as I observed) but simply didn't take care of this yet because I never saw any negative side effect beside the mere error message. Also I couldn't see any loss of functionality so far.
Regards,
Stefan

**Comment by Flyer53** [comment]
Fixed with v4.4.0
